cmd/kube-apiserver/app/aggregator.go

```diff
@@ -98,6 +98,7 @@ func createAggregatorServer(aggregatorConfig *aggregatorapiserver.Config, delega
 	}
 	autoRegistrationController := autoregister.NewAutoRegisterController(aggregatorServer.APIRegistrationInformers.Apiregistration().InternalVersion().APIServices(), apiRegistrationClient)
 	apiServices := apiServicesToRegister(delegateAPIServer, autoRegistrationController)
+	aggregatorServer.SetInternalAPIServices(apiServices)
```

Comment:
Doing this prevents a user from taking manual control of the APIServices later on and prevents any change by the end user, breaking the contract of how APIServices work.